### PR TITLE
Tick version to 2.1.1, rerender

### PR DESCRIPTION
**Pull request by pmlandwehr**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,16 +1,17 @@
 {%set name = "paramiko" %}
-{%set version = "1.18.1" %}
+{%set version = "2.1.1" %}
```
Comment (jakirkham) [inline recipe/meta.yaml]: They just released 2.1.2 today.

Comment (pmlandwehr) [reply recipe/meta.yaml]: Bah.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,16 +1,17 @@
 {%set name = "paramiko" %}
-{%set version = "1.18.1" %}
+{%set version = "2.1.2" %}
+{%set compress_type = "tar.gz" %}
 {%set hash_type = "sha256" %}
-{%set hash_val = "a17f693935ece50fdc647258d7e1d02f1499e534e9887c29cb02333695e907d8" %}
+{%set hash_val = "5fae49bed35e2e3d45c4f7b0db2d38b9ca626312d91119b3991d0ecf8125e310" %}
```
Comment (jakirkham) [inline recipe/meta.yaml:5]: Checksum doesn't seem to be correct.

Comment (jakirkham) [reply recipe/meta.yaml:5]: Resolved. This is correct for 2.1.2. Was only incorrect for 2.1.1.

**Comment by jakirkham** [comment]
Appears the requirements have changed. Now the requirements are...

```yaml
run:
  - cryptography >=1.1
  - pyasn1 >=0.1.7
```

No longer seeing `pycrypto` or `ecdsa`. So think we should drop them.

**Pull request merged**